src/main/java/io/frictionlessdata/tableschema/field/NumberField.java

```diff
@@ -52,10 +52,6 @@ public NumberField(String name, String format, String title, String description,
     public Number parseValue(String value, String format, Map<String, Object> options) throws InvalidCastException, ConstraintsException {
         String locValue = value.trim();
         try{
-            if (value.equalsIgnoreCase("null"))
-                return null;
-            if (value.length() == 0)
```

Comment:
Why would an empty field cause an exception instead of returning null? I understand that the spec talks about "a non-empty finite-length sequence of decimal digits", but does that really mean we cannot have empty values for numbers? I doubt it, but there is no definite answer in the spec

Reply:
Hi, 

thanks for the reply, hope you are fine. The question is:  is it correct to map the empty values ​​as numbers and not as strings?

Removing this two line I agree that an exception is raised (as for the others unmappable case for types) and it will handled as String. In the baseline, the empty value will have Number as default.